lib/tasks/scripts/message_patients.rb

```diff
@@ -0,0 +1,85 @@
+class MessagePatients
+  def self.call(*args)
+    new(*args).call
+  end
+
+  attr_reader :patients, :message, :whatsapp, :dryrun, :verbose, :report
+
+  def initialize(patients, message, whatsapp: true, dryrun: false, verbose: true)
```

Comment:
Let's change this instead to `channel: :whatsaap`, or `message_type: :whatsapp`?

Reply:
Ha, that's what I [started](https://github.com/simpledotorg/simple-server/pull/1099/commits/2d55c532f121b64ae54e4b715117bf750c684da3#diff-4b9785d5cf8fa1feff9c07408e1ce001R33) with and then reverted it to be this way because there were only two states so it made the conditional simpler. Fine with either way.

Reply:
Since this isn't a background job, where maintaining interface is super important, let's leave it as is 😄